pmlc/dialect/tile/transforms/pad_constraints.cc

```diff
@@ -0,0 +1,266 @@
+// Copyright 2019, Intel Corporation
+
+#include <algorithm>
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "mlir/Analysis/AffineStructures.h"
+#include "mlir/IR/AffineExprVisitor.h"
+#include "mlir/IR/AffineMap.h"
+#include "mlir/Pass/Pass.h"
+
+#include "pmlc/dialect/eltwise/ir/ops.h"
+#include "pmlc/dialect/tile/ir/ops.h"
+#include "pmlc/dialect/tile/transforms/padding.h"
+#include "pmlc/dialect/tile/transforms/pass_detail.h"
+
+namespace pmlc::dialect::tile {
+
+using mlir::AffineExpr;
+using mlir::AffineMap;
+
+namespace {
+
+bool validForPadding(AggregationKind agg, CombinationKind comb) {
```

Comment:
nit: call this `isValidForPadding` since it returns a `bool`

Reply:
Fixex.